Makefile

```diff
@@ -94,18 +94,28 @@ dev-js:
 	cd web && yarn install && yarn run start
 
 check-js:
+ifneq ($(CIRCLECI),true)
```

Comment:
maybe all our yarn commands should use a frozen lockfile unless you specifically say otherwise? wdyt?

Reply:
I'm happy to do that, I just wasn't sure how many folks were using `make` to do JavaScript tasks in their local development. If they are, such a change would break them.

But we can cross that bridge when we get to it. I'll make that change now.